ayekan/fluentbit/daemonset-config.yaml

```diff
@@ -0,0 +1,104 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: fluent-bit-config
+data:
+  fluent-bit.conf: |
+    [SERVICE]
+        Flush        1
+        Log_Level    debug
+
+    [INPUT]
+        Name              tail
+        Path              /var/log/containers/*.log
+        Parser            docker
+        Tag               kube.*
+        Mem_Buf_Limit     5MB
+        Skip_Long_Lines   On
+
+    [OUTPUT]
+        Name            opensearch
+        Match           kube.*
+        Host            ayekan-opensearch.opensearch
+        Port            9200
+        Logstash_Format On
+        Tls             On
+        Tls.Verify      Off
+  filers.conf: |
+    [FILTER]
+      Name                kubernetes
+      Match               kube.*
+      Kube_URL            https://kubernetes.default:443
```

Comment:
Does this work?  I think I've only seen `kubernetes.default.svc` used.

Reply:
It does, since it matches the scheme used in other deployments `<servicename>.<namespace>:<port>`. I used to think the `.svc` was needed as well, but in other deployments this scheme used to fail.